storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.cpp

```diff
@@ -747,8 +851,7 @@ RS_Status WriteColToRespBuff(const NdbRecAttr *attr, PKRResponse *response) {
     if (GetByteArray(attr, &data_start, &attr_bytes) != 0) {
       return RS_CLIENT_ERROR(ERROR_019);
     } else {
-      size_t encoded_str_size = boost::beast::detail::base64::encoded_size(attr_bytes);
-      char buffer[encoded_str_size];
+      char buffer[MAX_TUPLE_SIZE_IN_WORDS*4 + 2];
```

Comment:
possible buffer overflow. encoding the data increases the size of the data by 33%. the buffer size should be `math.ceil((4096/3)*4) = 5462`
Create a constant for it. If possible add a unit test for it